### ISSUE TITLE
HTTP Requests manpage refers to old() helper-function, but function is absent from Helper Function manpage

### ISSUE THREAD

**Issue by mmic-bjohnson** [laravel/framework#8509]
I'm reporting this inconsistency rather than submitting a pull request, mostly because it's not clear whether the `old()` function was simply omitted from the Helper Functions manpage inadvertently, or it was deprecated intentionally.

The specific nature of the problem is that the last example under http://laravel.com/docs/5.0/requests#old-input cites the `old()` helper-function, but there is no mention of this function at http://laravel.com/docs/5.0/helpers .

Calling `old()` within a Blade template doesn't seem to throw an error, whatever that's worth.

If someone is able to speak authoritatively as to the actual intent here, I'm happy to make the change and submit the pull request.

Also, why no Issue Tracker for the `docs` repository?

**Comment by GrahamCampbell** [comment]
We're open to pulls.

**Comment by franzliedke** [comment]
Do you even read these?`

> I'm reporting this inconsistency rather than submitting a pull request, mostly because it's not clear whether the old() function was simply omitted from the Helper Functions manpage inadvertently, or it was deprecated intentionally.

**Comment by mmic-bjohnson** [comment]
Thanks for jumping-in here, franzliedke. I was wondering the same.

Candidly, it makes no difference to me whether this gets fixed or not. I don't have time to dig into the source-code and review commit histories in an effort to make this determination myself (especially when a core contributor probably already knows whether `old()` was removed or was simply omitted from the manpage inadvertently).

Welp, I tried. :)

**Comment by GrahamCampbell** [comment]
@franzliedke We don't find issues saying "this is probably an issue" very helpful I'm afraid. If there is an issue, a pull request is the only thing of any use to us, or a short description/example of exactly what the issue is.

**Comment by GrahamCampbell** [comment]
> I'm happy to make the change and submit the pull request.

You don't need to ask permission to do that, infact, we'd encourage contributors not to. Sorry if this wasn't clear. I find that this way we can get things done more quikcly.

**Comment by mmic-bjohnson** [comment]
> If there is an issue, a pull request is the only thing of any use to us, or **a short description/example of exactly what the issue is**.

Isn't that what I provided?

TLDR: I'll just submit the pull request and let the developers worry about the whether the change is accurate or not.

I was hesitant to add the `old()` function to the Helpers manual page without consulting the developers because there are several other helper-functions that appear in the source-code but are not listed on the Helpers manual page (e.g., `logger()`). I wanted to ensure that their omission was not intentional before submitting an erroneous pull request.

But if it is more efficient for the developers to reject pull requests in such cases, then that is perfectly fine with me. In the future, I won't ask, I'll assume.

**Comment by GrahamCampbell** [comment]
> TLDR: I'll just submit the pull request and let the developers worry about the whether the change is accurate or not.

Thank you. :)

**Comment by mmic-bjohnson** [comment]
For whatever it's worth, this appears to have been fixed. The `old()` helper is now on the Helpers manpage (at least in 5.1).Thanks to whomever fixed it. :)
